yml/dp02_dc2.yaml

```diff
@@ -7090,692 +7090,692 @@ tables:
     "@id": "#DiaObject.decl"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Declination coordinate of the position of the diaObject at time radecTai
   - name: diaObjectId
     "@id": "#DiaObject.diaObjectId"
     datatype: long
     mysql:datatype: BIGINT
-    description:
+    description: Unique id.
   - name: gPSFluxChi2
     "@id": "#DiaObject.gPSFluxChi2"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Chi^2 statistic for the scatter of gPSFlux around gPSFluxMean
   - name: gPSFluxErrMean
     "@id": "#DiaObject.gPSFluxErrMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Mean of the diaSource PSF flux errors
   - name: gPSFluxLinearIntercept
     "@id": "#DiaObject.gPSFluxLinearIntercept"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: y-intercept of a linear model fit to diaSource PSF flux vs time
   - name: gPSFluxLinearSlope
     "@id": "#DiaObject.gPSFluxLinearSlope"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Slope of a linear model fit to diaSource PSF flux vs time
   - name: gPSFluxMAD
     "@id": "#DiaObject.gPSFluxMAD"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Median absolute deviation of diaSource PSF flux. Does not include scale factor for comparison to sigma
   - name: gPSFluxMax
     "@id": "#DiaObject.gPSFluxMax"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Maximum diaSource PSF flux
   - name: gPSFluxMaxSlope
     "@id": "#DiaObject.gPSFluxMaxSlope"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Maximum ratio of time ordered deltaFlux / deltaTime
   - name: gPSFluxMean
     "@id": "#DiaObject.gPSFluxMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Weighted mean of diaSource PSF flux
```

Comment:
Weighted by diaSource fluxErr, or what?

Reply:
How about  `Mean of individual diaSource PSF fluxes weighted by inverse square of the fluxErr`